src/lib/shady-render.ts

```diff
@@ -153,18 +158,22 @@ const prepareTemplateStyles =
       removeStylesFromLitTemplates(scopeName);
       // And then put the condensed style into the "root" template passed in as
       // `template`.
-      const content = template.element.content;
-      insertNodeIntoTemplate(template, condensedStyle, content.firstChild);
+      const content = templateElement.content;
+      if (template) {
+        insertNodeIntoTemplate(template, condensedStyle, content.firstChild);
+      } else {
+        content.insertBefore(condensedStyle, content.firstChild);
```

Comment:
Because of the comment below this, I think this case breaks support for `@apply` when using native Shadow DOM. That *might* be ok, but at the least we need to document it.

Reply:
We should be able to add a test that demonstrates this.

Reply:
I added the comment we were working on in offline discussion.